### PR TITLE
CLOUDP-239117: Increased resources limits for AKO container

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### All Submissions:

* [ ] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).
* [ ] Update docs/release-notes/release-notes.md if your changes should be included in the release notes for the next release.

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -45,8 +45,8 @@ spec:
           imagePullPolicy: Always
           resources:
             limits:
-              cpu: 500m
-              memory: 256Mi
+              cpu: 1000m
```
Comment (s-urbaniak) [inline config/manager/manager.yaml:48]: we shouldn't set cpu limits.

**Review by s-urbaniak — CHANGES_REQUESTED** [review]

File: config/manager/manager.yaml (hunk 1 above)
Comment (s-urbaniak) [reply config/manager/manager.yaml:48]: xref https://home.robusta.dev/blog/stop-using-cpu-limits

**Comment by s-urbaniak** [comment]
generally: I'd recommend running integration tests locally, using kind with a deployed Prometheus setup and use https://github.com/robusta-dev/krr to measure.

I can help here if needed.

**Comment by igor-karpukhin** [comment]
As per our conversation @s-urbaniak I updated the PR.

**Review by s-urbaniak — APPROVED** [review]

**Pull request merged**